app/src/components/createLearningJourney/SubmitButton.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useLJCreationContext } from "src/contexts/LJCreationContext";
+
+export default function SubmitButton({onClick}) {
+  const { selectedCourseDetails } = useLJCreationContext();
+
+  return (
+    <div className="flex-shrink-0 items-center mt-5 mr-auto">
+      {
+        Object.keys(selectedCourseDetails).length === 0
+          ? <button disabled className="relative inline-flex items-center rounded-md border border-transparent text-white bg-gray-300 hover:bg-gray-400 cursor-not-allowed bg-grey-500 px-4 py-2 text-sm font-medium shadow-sm" type="submit">
+            Disabled
```

Comment:
tbh i think q weird to name the button "Disabled", toggling the color should be enough. Can also delete the on-hover class.